include/xtensor-r/rcontainer.hpp

```diff
@@ -178,7 +178,7 @@ namespace xt
     {
         if (shape.size() != this->dimension() || !std::equal(std::begin(shape), std::end(shape), this->shape().cbegin()))
         {
-            derived_type tmp(std::forward<S>(shape));
+            derived_type tmp(xtl::forward_sequence<shape_type>(shape));
```

Comment:
I'm not sure about this line because if S is a reference and the same type, then isn't this casted to an rvalue (even though S is a reference), because we're not using the universal template `S&&` in the template of forward?
Or am I getting forwarding wrong?

Reply:
As discussed in person I think this is fine.